buildman/manager/ephemeral.py

```diff
@@ -523,8 +523,12 @@ def schedule(self, build_job):
 
             try:
                 execution_id = yield From(executor.start_builder(realm, token, build_uuid))
-            except:
-                logger.exception("Exception when starting builder for job: %s", build_uuid)
+            except exception as e:
```

Comment:
Can you also add a `logger.error` into the actual Kub start exception handler?

Reply:
Done